src/routers/UserRouter.ts

```diff
@@ -220,7 +220,7 @@ const getAdminRouter = (basePath: string = '/', routePath: string = '/management
       const isOwner = await authzService.isGroupOwner(req.identity.sub, groupId);
 
       const nestedGroupsFilter = isOwner
-        ? nestedGroups
+        ? nestedGroups.filter((r) => ['OWNER'].every((k) => !r.name.endsWith(k)))
         : nestedGroups.filter((r) => ['OWNER', 'ADMIN'].every((k) => !r.name.endsWith(k)));
```

Comment:
Can we extract this as a helper function on the `auth0-authz` service?

```
const nestedGroupsFilter = isOwner
        ? nestedGroups	        ? nestedGroups.filter((r) => ['OWNER'].every((k) => !r.name.endsWith(k)))
        : nestedGroups.filter((r) => ['OWNER', 'ADMIN'].every((k) => !r.name.endsWith(k)));
```

or change the existing `getNestedGroups` function to accept a optional filter by clause:
```
type Groups = 'OWNER' |  'ADMIN' | 'EDITOR' | 'VIEWER';
authzService.getNestedGroups(groupId: string, excludeGroups: Groups[] = [])
```